src/components/Header/index.js

```diff
@@ -21,6 +21,9 @@ const AuthenticatedHeaderOption = () => {
       <a className="link" href={PATHS.CLASS}>
         Classes
       </a>
+      <a className="link" href={PATHS.PARTNERS}>
+        Dashboard
```

Comment:
Rename to 'Partners'. Also, shouldn't this be only visible to the users who have admin related roles? Students should not be able to see this route. Can you confirm if this is the case?

Reply:
Yes, I confirmed this only admin can see this route. (Done)